routes/web.php

```diff
@@ -25,9 +25,7 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-})->name('welcome');
+Route::get('/', [TeamsController::class, 'getSponsors'])->name('welcome');
```

Comment:
I'd rather we have a `HomeController` or something, but I don't like the idea of the welcome page being from the teams controller.